__tests__/app.test.js

```diff
@@ -0,0 +1,29 @@
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const request = require('supertest')
+const db = require("../db/connection")
+const testData = require("../db/data/test-data/index")
+
+
+beforeEach(() => {
+    return seed(testData)
+    });
+
+  afterAll(() => {
+    db.end()
+  });
+
+describe("GET API/topics", () => {
+    test("Objects in the array should have the predefined keys", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({body}) => {
+            expect(body.length > 0).toBe(true)
```

Comment:
Really good to have this length check, remember though that we are using the test database for our data here, so we know exactly how many items are in this array, so could we be more specific with this length check number?